src/settings.py

```diff
@@ -44,6 +44,12 @@ class GlobalSettings:
     # in unit tests, make sure to pass in a value for `render_state_dpi` into
     # your call to utils.reset_config().
     render_state_dpi = 150
+    # If this is True, then we will not execute our final plan in simulation
+    # and we will say a task is solved successfully if we were able to find
+    # a plan. Note, we might solve the task even if the plan we found would
+    # not necessarily work in simulation (this is especially true of option
+    # models that don't model the simulator well).
```

Comment:
Add: "This can only be True if the approach is a subclass of BilevelPlanningApproach."